requirements.txt

```diff
@@ -13,5 +13,5 @@ redis<3.2
 requests==2.20.1
 sentry-sdk[flask]==0.7.8
 slackclient==1.3.0
-SQLAlchemy==1.2.14
+SQLAlchemy==1.3.1
```

Comment:
This version allows us to use JSON columns in SQLite, which is how we test locally. FTW!